tox.ini

```diff
@@ -91,4 +73,4 @@ conda_deps =
     zeep
 conda_channels = conda-forge
 commands =
-    pytest -m "not figure" --cov=./sunpy {posargs}
+    pytest -v --pyargs sunpy -m "not figure" --cov=sunpy --junitxml=junit/test-results.xml --cov-report=xml --cov-report=html {posargs}
```

Comment:
I am not a fan of putting the junit and the xml cov report in the tox.ini I think they are better off being a thing specific to Azure?

Reply:
K